### DISCUSSION TITLE
Read initial values from Conversion

### DISCUSSION THREAD

**Discussion by Porley** [danielhrisca/asammdf#500] [Q&A]
Hello,

first, I would like to thank Daniel for providing such a wonderful piece of software. I am using asammdf for roughly six month now, and I love it! I am using version 5.23.2.
Now I came across a topic from which I don't know whether there is an easy way to access the information I need. I would like to extract the initial value for a signal. With Vector Mdf Validator, I recognize that the information is part of the signal (see screenshot); however, I could not enter it yet.
Is there a way to extract this information? How would I do that? Any help will be highly appreciated :-)
![2021-03-25 08_32_20-Vector MDF Validator](https://user-images.githubusercontent.com/72650178/112437029-c6f36880-8d46-11eb-9109-6e0786f404c8.png)

**Comment by danielhrisca** [comment]
@Porley that is a CANape specific feature. For now you can just see the extra bytes after the normal LinearConversion block using the attribute `conv.CANapeHiddenExtra`

**Reply by Porley** [reply]
Wow, that was fast :-) Thanks a lot for the reply. I will try this out.
Thanks again, Porley
